code/mob/living/silicon/robot.dm

```diff
@@ -70,6 +70,8 @@
 	var/alohamaton_skin = 0 // for the bank purchase
 	var/metalman_skin = 0	//mbc : i'm getting tired of copypasting this, i promise to fix this somehow next time i add a cyborg skin ok
 	var/glitchy_speak = 0
+	///how many flips with our head exposed have we done?
```

Comment:
I don't really like this solution. If I flip 4 times in a round, blammo.
I'd prefer a status effect or something that is like "loose brain".